robocorp-code/vscode-client/src/extension.ts

```diff
@@ -483,7 +483,7 @@ export async function activate(context: ExtensionContext) {
             roboCommands.ROBOCORP_RENAME_RESOURCE_IN_ROBOT_CONTENT_VIEW,
             renameResourceInRobotContentTree
         );
-        commands.registerCommand(roboCommands.ROBOCORP_UPDATE_LAUNCH_ENV, updateLaunchEnvironment);
```

Comment:
The idea would be commenting this line out (the `updateLaunchEnv` should return the new env, so, if it returns something empty the implementation may be broken -- but if the command is not there the callers should be protected for that situation already).

Reply:
Commented out the line.